client/clientservice/src/request_service.cpp

```diff
@@ -47,27 +47,35 @@ Status RequestServiceImpl::Send(ServerContext* context, const Request* proto_req
       LOG_INFO(logger_, "Send returned error");
       switch (std::get<uint32_t>(send_result)) {
         case (static_cast<uint32_t>(bftEngine::OperationResult::INVALID_REQUEST)):
+          LOG_INFO(logger_, "Returned INVALID_ARGUMENT");
```

Comment:
Replace by "Request failed with INVALID_ARGUMENT error for cid=" << req_config.correlation_id
etc.

Reply:
done